completions/timeout

```diff
@@ -2,17 +2,17 @@
 
 _timeout()
 {
-    local cur prev words cword split comp_args i found=false
+    local cur prev words cword split comp_args i found=""
     _comp_initialize -s -- "$@" || return
 
     local noargopts='!(-*|*[ks]*)'
     for ((i = 1; i <= COMP_CWORD; i++)); do
         if [[ ${COMP_WORDS[i]} != -* && ${COMP_WORDS[i - 1]} != = ]]; then
-            if $found; then
-                _comp_command_offset $i
+            if [[ $found ]]; then
+                _comp_command_offset "$i"
```

Comment:
Hmm, I suppose this means we should be quoting all arguments containing variables to any commands or functions we invoke? This PR does not address them all throughout the codebase, but we can take care of them in followup PR's.

I'm starting to feel that we should write up some rules about all this stuff in our docs somewhere, perhaps styleguide.md. And it would be great if there was some tooling that would help by flagging or fixing undesired uses. I wonder if shellcheck or shfmt would accept some of that stuff.

Reply:
This is also dropped together with 07cec4411a75c64f8a695338e51d9226f78f3fce. Sorry for the noise.

But thank you for your consideration and comments. Maybe we could leave an issue to later address quoting matter.

Reply:
#932